packages/data-stores/src/site/actions.ts

```diff
@@ -392,8 +415,11 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 			method: 'POST',
 		} );
 
+		if ( styleVariation?.slug === DEFAULT_GLOBAL_STYLES_VARIATION_SLUG ) {
+			yield* resetGlobalStyles( siteSlug, activatedTheme.stylesheet, activatedTheme );
+		}
 		// @todo Always use the global styles for consistency
```

Comment:
@arthur791004 do you happen to remember what this `// @todo` comment is referring to?

If you scroll down to line `443`, it looks like it _is_ always using the global styles by calling:
```ts
		if ( globalStyles ) {
			yield* setGlobalStyles( siteSlug, activatedTheme.stylesheet, globalStyles, activatedTheme );
		}
```

But maybe I'm missing something?

The comment was added in #78313 when working on font selection, but it seems the font selector is no longer accessible in onboarding.

Reply:
The font selector is still accessible - it appears for first-party block themes that don't have any existing style variations. Try Impressionist or Pomme for example.

Reply:
Thanks for that! I've tested the PR with Impressionist and confirmed setting a default style variation clears the "upgrade plan" message from the launchpad as expected.

Reply:
However overriding a premium block style variation with the default "Impressionist" theme doesn't clear the message from the launchpad 😮 So that needs to be fixed.

Reply:
Sorry for the late reply. If I remember correctly, it means we pass the style variation we want as `globalStyles` option so we don't need the condition below anymore.